conjure-java-core/src/integrationInput/java/com/palantir/product/EnumExample.java

```diff
@@ -35,6 +37,8 @@
     @Deprecated
     public static final EnumExample ONE_HUNDRED = new EnumExample(Value.ONE_HUNDRED, "ONE_HUNDRED");
 
+    private static final List<EnumExample> VALUES = ImmutableList.of(ONE, TWO, ONE_HUNDRED);
```

Comment:
What happens if there's an enum value with name `VALUES`? perhaps we should use `values` here instead?

Reply:
Ah, I checked and saw there was a dependency on guava, so preferred immutable over unmodifiable. I've switched now.

Reply:
Good call, done.

Reply:
or `_VALUES`